.github/workflows/release-sync.yml

```diff
@@ -0,0 +1,157 @@
+name: Sync Release with Master
+
+# Triggered when a new version is released publicly
+on:
+  workflow_dispatch:
+  release:
+    types: [ released ]
+
+jobs:
+  create-sync-pull-request:
+    name: Create release sync PR
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.RELEASE_TOKEN }}
+      ORG_NAME: HathorNetwork
+      PROJECT_NUMBER: 15
+      COLUMN_NAME: "In Progress (WIP)"
+
+    steps:
+      - name: Checkout
+        id: checkout
+        # https://github.com/actions/checkout/releases/tag/v4.1.6
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29
+
+      - name: Create Pull Request
+        id: create_pull
+        run: |
+          # The assignee will be the user that manually published the release
+          PR_URL=$(gh pr create \
+            --title "chore: [${{ github.ref_name }}] Merge release into master" \
+            --body "Automated PR to merge `release` branch into `master` based on release event." \
+            --base "master" \
+            --head "release" \
+            --assignee "@me")
```

Comment:
Will this `@me` work? 

And can we set one of the reviewers to be `@me` as well? If the PR is automatically created, we can have this dev as one of the reviewers, I think

Reply:
Yes, `@me` works for this command, but unfortunately can't be set as a reviewer.

This is because the actor who triggered the action will also be the one set as the _creator_ of the PR, and the creator can't be set as a reviewer.

A solution to this would be to build a dedicated bot like @dependabot , that would create the PR and then set the assignee and reviewer to the actor who triggered the release.